x-pack/filebeat/module/zeek/weird/ingest/pipeline.yml

```diff
@@ -1,8 +1,11 @@
 description: Pipeline for normalizing Zeek weird.log
 processors:
 - set:
-    field: event.created
+    field: event.ingested
     value: '{{_ingest.timestamp}}'
+- set:
+    field: event.created
+    value: '{{@timestamp}}'
```

Comment:
these almost make me wonder if we should add `event.created` by default to the `@timestamp` populating code.

Reply:
> Zeek already had `event.created`, but it was being set to `_ingest.timestamp` so I changed it to `@timestamp`. The diff looks like I added a new event.created field but really I changed it.
> 
> I do think that for consistency we should add `event.created` and also `event.original` to every module. But that's probably best to do in separate pull requests.

@andrewstucki @andrewkroh 
So the ECS docs have this for event.created:
```
In case the two timestamps are identical, @timestamp should be used.
```
I'm wondering if we should be setting event.created & @timestamp to the same value?